### PR TITLE
feat: bump stylus-sdk-rs version to v0.5.2

### PR DESCRIPTION
**Pull request by alexfertel**

### CODE REVIEW

File: contracts/src/token/erc721/mod.rs [hunk 1]
```diff
@@ -1092,7 +1088,7 @@ impl Erc721 {
                 operator,
                 from,
                 token_id,
-                data.to_vec(),
+                data.to_vec().into(),
```
Comment (bidzyyys) [inline contracts/src/token/erc721/mod.rs:1091]: Curious if we can use just `data.into()`.

**Review by bidzyyys — APPROVED** [review]
LGTM! 🚀

File: contracts/src/token/erc721/mod.rs (hunk 1 above)
Comment (alexfertel) [reply contracts/src/token/erc721/mod.rs:1091]: We can't sadly, there's some weirdness here in the new types it seems :(

**Comment by alexfertel** [comment]
Leaving as a draft until we hear back from the Offchain Labs team.

**Comment by bidzyyys** [comment]
There is a new version, `v0.5.2` maybe worth trying?

**Comment by alexfertel** [comment]
@bidzyyys still broken for the same reason :(

**Pull request closed without merging**